doc/developer-guide/plugins/hooks-and-transactions/ssl-hooks.en.rst

```diff
@@ -84,11 +84,28 @@ handshake processing will not proceed until :c:func:`TSSslVConnReenable()` is ca
 It may be useful to delay the TLS handshake processing if other resources must be consulted to select or create
 a certificate.
 
+TS_VCONN_OUTBOUND_START_HOOK
+----------------------------
+
+This hook is invoked after ATS has connected to the upstream server and before the SSL handshake has started.  This gives the plugin the option of 
+overriding the default SSL connections on the SSL object.
```

Comment:
"default SSL connections"? Do you mean "connection options"?

Reply:
You are correct.  Will update.